docs/docs/security.md

```diff
@@ -49,14 +49,8 @@ provided after each network message from a client is delivered, the enclave can
 point. Then, stored messages from the clients can be replayed back to the enclave in different orders, or with some
 messages dropped.
 
-Such attacks are called memento/rewind/replay attacks. You need to consider this type of attack while designing your
-app.
-
-!!!Warning
-
-    A full discussion of memento attacks is beyond the scope of this document. The Conclave project strives to provide
-    you with high-level protocol constructs that take them into account, but when writing your own protocols, you should
-    consider what happens when messages can be re-ordered and re-tried by the host.
+Such attacks are called memento/rewind/replay attacks.
+You can use Conclave's [persistent map](persistence.md#PersistentMap) to mitigate rewind attacks.
```

Comment:
The link isn't correct. I believe the heading link should be `#persistent-map`

Reply:
Corrected.